src/Worker/Core/DurableTaskWorkerOptions.cs

```diff
@@ -83,6 +83,21 @@ public DataConverter DataConverter
     /// </remarks>
     public TimeSpan MaximumTimerInterval { get; set; } = TimeSpan.FromDays(3);
 
+    /// <summary>
+    /// Gets or sets the maximum number of concurrent activity work items that can be processed by the worker.
+    /// </summary>
+    public int MaximumConcurrentActivityWorkItems { get; set; } = 100 * Environment.ProcessorCount;
```

Comment:
What do you think of grouping these into a sub-class?

``` Csharp
public ConcurrencyOptions Concurrency { get; } = new();

// can be a nested class
public class ConcurrencyOptions
{
    public int MaximumActivityWorkItems { get; set; }

    // and the other fields.
}
```

Reply:
I'm fine either way, but a flat list seemed simple enough.

Reply:
I am a fan of nesting relevant options together. Mostly for organization for both ourselves as maintainers, and for users for discovering all concurrency related config together in one spot (and not having unrelated config mixed in).

But I am not so adamant about it to block a PR on that. Just something for you to consider.

Reply:
I went ahead and added it. I can see this being useful over time as we add more options.